Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+
+RUN apk add --update ca-certificates \
+    && rm -rf /var/cache/apk/*
```

Comment:
I just spread the knowledge, there is a newly added flag. Not sure if this is better.